src/containers/Tenant/utils/controls.tsx

```diff
@@ -71,7 +74,7 @@ export const getSchemaControls =
             column_table: openPreview,
 
             index_table: undefined,
-            topic: undefined,
+            topic: isTopicPreviewAvailable && !meta?.subType ? openPreview : undefined,
```

Comment:
You should also turn off preview in preview table component. Currently there are not preview button for topic, but if I open preview and then try to navigate from table to cdc impl topic, I will receive an error

<img width="1390" alt="Screenshot 2025-05-21 at 15 50 47" src="https://github.com/user-attachments/assets/e81b64bb-d734-4663-9009-9c30d45163c2" />